tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>I'd remove this here and just repeat the message in the below sections to ensure people get it, given how critical it is. </p><p> </p><p>I'd rephrase it to:</p><p> </p><p>"You must pass a pre-trained model with acceptable accuracy to this API. Training from scratch with clustering results in subpar accuracy results".</p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>